app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PasswordResetsController < ApplicationController
+  require_unauthenticated_access
```

Comment:
you’ll need to also change this one:
```suggestion
  allow_unauthenticated_access
```